### PR TITLE
[MCC-131207] update readme

### PR DESCRIPTION
**Pull request by csavage-mdsol**
Updates the readme with the usage example from current Farscape

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # CrichtonTestService
 
-TODO: Write a gem description
```
Comment (yzhangmedidata) [inline README.md:3]: Is the lack of description intentional

Comment (csavage-mdsol) [inline README.md:3]: lack yes, removing todo no.  Will fix.

Comment (yzhangmedidata) [inline README.md:3]: Ah cool

File: README.md [hunk 2]
```diff
@@ -18,7 +18,28 @@ Or install it yourself as:
...
+  }
+  WebMock.disable!
+  $crichton_test_service_rails_pid = CrichtonTestService.spawn_rails_process!(RAILS_PORT)
+end
+
+config.after(:suite) do
+  Process.kill(:INT, $crichton_test_service_rails_pid)
+end
+```
+This code in a ``` before(:suite) ``` hook first captures the old behavior for handling INT, ensures
```
Comment (yzhangmedidata) [inline README.md]: Maybe a note for what <code>INT</code> means

Comment (csavage-mdsol) [inline README.md]: Done

**Pull request merged**